return.h

```diff
@@ -117,26 +117,16 @@
 /*************************************/
 
 /*
- * For DEC Mips machines running Ultrix
+ * For Mips machines running Linux
```

Comment:
Is there some way to detect Mips running Linux better than just __mips?

Reply:
Nope, the only "standard" way is to check for `__mips` or `__mips__`(cf. https://sourceforge.net/p/predef/wiki/Architectures or https://wiki.debian.org/ArchitectureSpecificsMemo)